extensions/interactions/FractionInput/directives/fraction_input_interaction_directive.html

```diff
@@ -1,7 +1,7 @@
 <form ng-disabled="FractionInputForm.$invalid" ng-submit="submitAnswer(answer)" role="form" class="form-horizontal">
   <!-- This ng-form is needed for submitting answer on hitting enter. -->
   <ng-form name="FractionInputForm">
-    <input type="text" name="answer" ng-model="answer" class="form-control"
+    <input style="width:100%;" type="text" name="answer" ng-model="answer" class="form-control"
```

Comment:
This should be somewhere in the CSS style tag below instead (see lines 14-23).

Reply:
@seanlip When putting the styling in the style tag, it gets overwritten and couldn't be used there.

Reply:
Hi @WickedBrat I think it may work if you make a more specific css rule such as `.form-control.oppia-fraction-input` or similar.

Reply:
Hi @tjiang11, the css rule is applied when I add it but is overwritten by that of third-party.css, so that's the reason the defining separately in style won't work. I did try what you suggested but didn't work. If we have to use it in style tag, then we'll have to use ```!important```.

Reply:
No, @tjiang11's approach here is the correct one. There should be a way to make the CSS rule even more specific so that it precedes the third-party CSS. (I suggest reading up on specificity in CSS to make sure you understand how priority is determined.)

(Note that using !important basically hijacks the specificity order, and is therefore discouraged.)

Reply:
Oh! I didn't know about this @seanlip! I did the changes. Just out of curiosity, why is not suggested to use inline css?